src/services/signalr.configuration.ts

```diff
@@ -19,13 +19,17 @@ export class SignalRConfiguration {
      /** Allows withCredentials. This flag can be used to suppport CORS */
     public withCredentials: boolean;
 
+    /**Allows you to specify transport. You can specify a fallback order if you wan't to try specific transports in order. */
+    public transport?: any;
```

Comment:
Is this of type string? array of strings? Can you explain the possible values you are expecting